src/Commands/ConfigCommand.php

```diff
@@ -37,10 +37,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $newContent = json_encode($config, JSON_PRETTY_PRINT);
         if (file_exists($configPath) && $newContent === file_get_contents($configPath)) {
             $output->writeln(sprintf('No changes to %s file.', realpath($configPath)));
-            return;
+            return null;
         }
 
         file_put_contents($configPath, $newContent);
         $output->writeln(sprintf('<info>%s file written.</info>', realpath($configPath)));
+
+        return null;
```

Comment:
Why that?

Reply:
See the description. It's absolutely not necessary, but this overrides a function that returns `int|null`. If `: ?int` were actually in the signature, PHP would demand and explicit `return null`. So this is future-proofing, and harmless.

Reply:
Didn't notice it's overriding, fine then.

Reply:
Isn‘t the null return implicit in this case. The amount of warnings is starting to remond of of a java ide 😳

Reply:
Yup, you're right. Psalm shouldn't treat the function [docblock return type](https://github.com/symfony/console/blob/275c56f48261a67ab0c09bd445428e4b08bbe4a2/Command/Command.php#L145-L162) like it would a return typehint.

I've fixed in https://github.com/vimeo/psalm/commit/b373d6093a9a537836205cfc7de19140bfad08c3. This `return null` can stay, I think, if you ever decide to add an explicit `?int` to this function.